server/src/main/java/umm3601/Server.java

```diff
@@ -67,7 +67,8 @@ public static void main(String[] args) {
     // List products, filtered using query params
     server.get("/api/products", productController::getAllProducts);
 
-    //server.get("/api/products/group", productController::groupProductsByCategory);
+    // List the products grouped by their category
+    server.get("/api/products/group", productController::groupProductsByCategory);
```

Comment:
Is `group` a great name/term here? We could group a lot of different ways. Would `byCategory` or `by-category` or `/api/products-by-category` or some such be better?

Reply:
We should make an issue to "deprecate"/remove `/api/products/group` (or whatever it's called) and just have `/api/products` return the grouped result.